akka-docs/rst/java/code/docs/camel/CustomRouteTestBase.java

```diff
@@ -11,11 +11,14 @@
   public void customRoute() throws Exception{
     //#CustomRoute
     ActorSystem system = ActorSystem.create("some-system");
-    Camel camel = CamelExtension.get(system);
-    ActorRef responder = system.actorOf(Props.create(Responder.class), "TestResponder");
-    camel.context().addRoutes(new CustomRouteBuilder(responder));
-    //#CustomRoute
-    system.stop(responder);
-    JavaTestKit.shutdownActorSystem(system);
+    try {
+      Camel camel = CamelExtension.get(system);
+      ActorRef responder = system.actorOf(Props.create(Responder.class), "TestResponder");
+      camel.context().addRoutes(new CustomRouteBuilder(responder));
+      //#CustomRoute
+      system.stop(responder);
```

Comment:
Why does this matter if the next things we do is to terminate the entire actor system?